src/jit/liveness.cpp

```diff
@@ -1323,6 +1322,13 @@ class LiveVarAnalysis
 
 void Compiler::fgLiveVarAnalysis(bool updateInternalOnly)
 {
+#if CAN_DISABLE_DFA
```

Comment:
Currently `CAN_DISABLE_DFA` is never false. We could probably just remove the symbol entirely--I only used it in because these changes depend on the minopts-only code in `fgPerBlockLocalVarLiveness` that is also conditional on this symbol.

I agree that we should keep this consistent with the allocator's decisions, though I think that minopts should remain a part of the condition as liveness is also used by DSE and the emitter. I think the appropriate condition under which to disable these operations is `if (opts.MinOpts() && !enregisterLocalVars)`.